py/sigver/verify.py

```diff
@@ -72,11 +63,19 @@ def main():
     LOG.info (json.dumps(data))
 
     try:
-        response = requests.post(URL, headers=headers, data=json.dumps(data), timeout=20)
+        status_code=0
+        count = 3
+        while (status_code!=200 and count > 0):
+            response = requests.post(URL, headers=headers, data=json.dumps(data), timeout=20)
+            status_code=response.status_code
+            if status_code!=200:
+                count -= 1
+                LOG.error("Error %i Response: %s", status_code, response.text)
```

Comment:
I assume `response.text` is empty string for most errors. You might want to change this log entry to something like: 
```
LOG.error("Error %i Response text: '%s'", status_code, response.text)
```